charts/wga/values.yaml

```diff
@@ -7,10 +7,23 @@
 ## @param global.imageRegistry Global Docker image registry
 global:
   imageRegistry: "ghcr.io/kraudcloud"
+  privateImageRegistry: "ctr.0x.pt"
 
 ## @section Common parameters
 ##
 
+welcome:
+  image: 
+    name: "wga-welcome"
+    tag: "main"
+    pullPolicy: ""
+
+  service:
```

Comment:
whats this for?

Reply:
copied from the web config down below, kept it because it actually might be useful in the future, leaves a defined spot for it

Reply:
no. its not supposed to have its own service